Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/PlayButton.swift

```diff
@@ -71,7 +82,18 @@ open class PlayButton: MediaControl.Element {
         show()
         changeToPlayIcon()
     }
+
+    private func onComplete() {
+        show()
+        changeToReplayIcon()
+    }
     
+    private func onWillSeek(_ position: Double) {
+        guard let playback = activePlayback, position != playback.duration else { return }
+        show()
+        canShowPlayIcon ? changeToPlayIcon() : changeToPauseIcon()
```

Comment:
The method show is already present on `changeIcons`
```suggestion
        canShowPlayIcon ? changeToPlayIcon() : changeToPauseIcon()
```

Reply:
I think `show` is not called inside those methods but rather on the `onPause` and `onStop` 🤔